tensorflow/core/kernels/dml_cwise_ops.cc

```diff
@@ -713,6 +713,8 @@ REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(RsqrtGrad, y * (-0.5f * x) * (x * x),
                                            kNchwDimensionCount)
 REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(ReciprocalGrad, -y* x* x,
                                            kNchwDimensionCount)
+REGISTER_DML_COMPOSITE_BINARY_FLOAT_KERNEL(InvGrad, -y* x* x,
```

Comment:
```suggestion
-y * x * x
```

Reply:
Clang format formats it that way, probably because it's used in a macro and it's missing context, so it assumes they are pointers.

Reply:
(🤦‍♀️ Stupid autoformatters. I wish GitHub had a "sad" reaction.)